Books.js

```diff
@@ -29,12 +30,12 @@ export default class Books {
 
   booksDisplay = () => {
     this.booksArray.forEach((book, index) => {
-      this.booksHtml = (title, author) => `    <h2 id="book-title">${title}</h2>
-      <h3 id="book-author">${author}</h3>
+      this.booksHtml = (title, author) => `  <div class="book-container">  
+      <h3 id="book-author" class = "book-author">"${title}" by ${author}</h3>
       <button type="button" class="book-btn" id="book-btn">Remove</button>
-      <hr>`;
+      </div>`;
       const htmlToAdd = this.booksHtml(book.title, book.author);
-      this.booksList.insertAdjacentHTML('afterbegin', htmlToAdd);
+      this.allBooks.insertAdjacentHTML('afterend', htmlToAdd);
       this.removeBtn = document.getElementById('book-btn');
       this.removeBtn.onclick = () => this.removeBook(index);
     });
```

Comment:
- [ ] When I click the remove button, the book item I intend to remove doesn't delete from the `localStorage` and nothing displays on the UI after the action. However, I noticed that when I add a book after clicking the removing button, the UI displays the old books but the new book is not added. Kindly fix this bug 👍 😎 

![chrome-capture-2023-2-21](https://user-images.githubusercontent.com/77106962/226653980-e15285a5-bf37-42f7-b123-9b5bfe998520.gif)